### PR TITLE
implement the "--localfile_abspath" command line switch

### PR DESCRIPTION
**Pull request by MarkusTeufelberger**
See #743

### CODE REVIEW

File: nbviewer/app.py [hunk 1]
```diff
@@ -281,6 +282,7 @@ def init_options():
     define("debug", default=False, help="run in debug mode", type=bool)
     define("no_cache", default=False, help="Do not cache results", type=bool)
     define("localfiles", default="", help="Allow to serve local files under /localfile/* this can be a security risk", type=str)
+    define("localfile_abspath", default=False, help="Use the absolute instead of the real path for local files, don't dereference symbolic links", type=bool)
```
Comment (parente) [inline nbviewer/app.py]: Minor suggestion: Maybe name the parameter `localfile_follow_symlinks`. Personally, I always need to google to remember which one of realpath or abspath dereferences the symlinks.

Comment (MarkusTeufelberger) [reply nbviewer/app.py]: Nice suggestion, I'm not great at naming stuff. ;-)

Will do.

File: nbviewer/providers/local/handlers.py [hunk 2]
```diff
@@ -37,7 +37,10 @@ class LocalFileHandler(RenderingHandler):
 
     @property
     def localfile_path(self):
-        return os.path.realpath(self.settings.get('localfile_path', ''))
+        if self.settings.get('localfile_follow_symlinks'):
+            return os.path.abspath(self.settings.get('localfile_path', ''))
+        else:
+            return os.path.realpath(self.settings.get('localfile_path', ''))
```
Comment (parente) [inline nbviewer/providers/local/handlers.py]: I think the logic here is inverted. `os.path.realpath` resolves (follows) symlinks to their targets (https://docs.python.org/3.6/library/os.path.html#os.path.realpath). `os.path.abspath` just gets the absolute path of the symlink itself. I think these need to be flipped, along with the code blocks below too.

Comment (MarkusTeufelberger) [reply nbviewer/providers/local/handlers.py]: Ah, I see it now, that was an oversight on my end in the option renaming. Thanks for the review!

**Comment by parente** [comment]
Thanks for the PRs @MarkusTeufelberger! 🍰

**Pull request merged**